src/light.jl

```diff
@@ -181,6 +181,11 @@ function out_transitions(A::LightAutomaton{GT, ET}, s) where {GT, ET}
     LightTransitionIterator(A, edges)
 end
 
+function get_transitions(A::LightAutomaton{GT, ET}, q, r) where {GT, ET}
+    e = edge_object(A, q, r)
+    LightTransitionIterator(A, [e])
```

Comment:
`[e]` could be a FillArray since it is constant. That would prevent the allocation needed to create the array